content/docs/guides/neon-authorize-future.md

```diff
@@ -0,0 +1,53 @@
+---
+title: The future for Neon Authorize
+subtitle: Our vision for what comes next
+enableTableOfContents: true
+---
+
+[Neon Authorize](/docs/guides/neon-authorize) is an exciting feature, and we want to build more on top of it.
+
+Here's what we'd like to work on next — in no particular order (unless stated otherwise). If you see something you like, or something you'd like to see but don't, let us know in [Discord](https://discord.com/channels/1176467419317940276/1176788564890112042)!
+
+## Accepting JWTs for TCP and WebSocket connections
+
+We want Neon Authorize to work over any type of connection: TCP, HTTP, or WebSockets. This is probably our highest priority. Accepting JWTs over TCP (in the password field for the Postgres role) would mean that any Postgres SDK in any programming language could support Neon Authorize. We will build this.
+
+## Custom roles
+
+We aim to introduce custom roles for Neon Authorize, beyond the "authenticated" and "anonymous" roles we have now. We're also exploring the idea of adding "Neon-managed", which would be managed by our platform and included by default in every branch and read replica.
+
+## JWT audience checks
+
+The `aud` field in a JWT typically defines the token's intended recipients. In the future, we want to allow users to customize the expected `aud` field in the JWTs they use with Neon Authorize, providing greater flexibility and control.
+
+## Multi-tenancy through Neon Authorize
+
+We want to support multi-tenancy with Neon Authorize, allowing users to configure JWTs that trigger Neon to route database requests to different Neon instances.
```

Comment:
"We've discussed supporting multi-tenancy via Neon Authorize, allowing users to configure how their JWTs should trigger Neon to route database requests to different Neon instances"